api/envoy/config/metrics/v3/stats.proto

```diff
@@ -406,3 +406,32 @@ message HystrixSink {
   // <https://github.com/Netflix/Hystrix/wiki/Metrics-and-Monitoring#hystrixrollingnumber>`_.
   int64 num_buckets = 1;
 }
+
+// Stats configuration proto schema for *envoy.stat_sinks.graphite_statsd* sink.
+// The sink emits stats with `Graphite <https://graphite.readthedocs.io/en/latest/tags.html>`_
+// compatible tags. Tags are configurable via :ref:`StatsConfig
+// <envoy_api_msg_config.metrics.v3.StatsConfig>`.
+// [#extension: envoy.stat_sinks.graphite_statsd]
+message GraphiteStatsdSink {
+  reserved 2;
+
+  oneof statsd_specifier {
```

Comment:
Why this need to be oneof, will there be more specifier later?

Reply:
Yes, TCP might become supported too in the future.